### ISSUE TITLE
Create cluster cmd fails

### ISSUE THREAD

**Issue by Bonczidai** [docker/swarm#1480]
Hi folks,

problem is that the simplest Swarm command fails: `docker run --rm swarm create`
I've tried it on DigitalOcean droplets and CircleCI build (note that, the command was working fine before).
Consider a CoreOS machine in DigitalOcean:

```
core@test-1 ~ $ docker run --rm swarm create
dial tcp 50.16.209.231:5452: getsockopt: connection timed out
```

version:
```
core@test-1 ~ $ docker version
Client:
 Version:      1.8.3
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   cedd534-dirty
 Built:        Tue Dec  1 02:00:58 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.8.3
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   cedd534-dirty
 Built:        Tue Dec  1 02:00:58 UTC 2015
 OS/Arch:      linux/amd64
```

Any ideas why this could happen?

Thanks,
Levi

**Comment by johnjelinek** [comment]
Duplicate of #1482

**Comment by aluzzardi** [comment]
Thanks for reporting - it's fixed now and we'll give more details in #1482 since more people commented on that one.
